setup.py

```diff
@@ -81,7 +81,7 @@ def get_version(file_path):
             'recommender = recommender:RecommenderXBlock',
         ]
     },
-    package_data=package_data("recommender", ["static", "templates", "translations"]),
+    package_data=package_data("recommender", ["static", "templates", "translations", "conf"]),
```

Comment:
It's not clear to me why `"conf"` is being added here. @shadinaif could you provide context?

Reply:
Your right, it has no meaning. Removed 👍🏼